src/emulator/controller.ts

```diff
@@ -971,7 +972,7 @@ export async function exportEmulatorData(exportPath: string, options: any, initi
 
     const prompt = await promptOnce({
```

Comment:
Optional - consider using [confirm](https://github.com/firebase/firebase-tools/blob/master/src/prompt.ts#L104) here instead - it already has some of this logic for handling force/non-interactive baked in